notebooks/official/model_evaluation/custom_tabular_regression_model_evaluation.ipynb

```diff
@@ -0,0 +1,2261 @@
+{
```

Comment:
<p>Since feature attributions are optional, what if the user didn't want to configure it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/942/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='103'/>

Reply:
<p>In "Upload the model" section added this-&gt; **If you want to configure explanations for the model, set <code>explanation_parameters</code>,&nbsp;<code>explanation_metadata</code>  arameters. Else do not set them.**</p><p> </p>

Reply:
<h3>Updated Get the Feature Attributions to below text</h3><h3><br></h3><h3><br></h3><h3>Get the Feature Attributions (Optional)</h3><p><strong>If you have configured explanations for the model, run below cell. Else skip below cell</strong></p>